src/components/GeoMap.js

```diff
@@ -530,16 +530,17 @@ export default class GeoMap {
   }
 
   exportSvg() {
+    if (!saveAs) {
+      throw new Error('MapTable: Missing FileSaver.js library');
```

Comment:
if you don't have `FileSaver.min.js` included I don't hit this line.  Instead in the console i see:

`Uncaught ReferenceError: saveAs is not definedexportSvg @ GeoMap.js:533`